app/routes/__index/certificate/information.tsx

```diff
@@ -0,0 +1,182 @@
+                and is issued by a trusted third party called a certification authority. For more
+                information&nbsp;
+                <Link
+                  color="brand.500"
+                  href="https://www.cloudflare.com/learning/ssl/what-is-an-ssl-certificate/"
+                >
+                  click here
+                </Link>
+              </Text>
+            </TabPanel>
+            <TabPanel>
+              <Text>
+                Used to decrypt data that has been encrypted with the corresponding public key, and
+                to digitally sign documents or messages. For more information&nbsp;
+                <Link
+                  color="brand.500"
+                  href="https://www.cloudflare.com/learning/ssl/what-is-an-ssl-certificate/"
+                >
+                  click here
+                </Link>
+              </Text>
+            </TabPanel>
+            <TabPanel>
+              <Text>
+                A set of certificates that are used to verify the authenticity of an SSL/TLS
+                certificate. They serve as a bridge between the SSL/TLS certificate and the root
+                Certificate Authority (CA) to establish a chain of trust, ensuring that the SSL/TLS
+                certificate is valid and secure. For more information&nbsp;
+                <Link
+                  color="brand.500"
+                  href="https://support.dnsimple.com/articles/what-is-ssl-certificate-chain/"
+                >
+                  click here
+                </Link>
+              </Text>
+            </TabPanel>
+            <TabPanel>
+              A file that combines an SSL certificate (Public Certificate) with its intermediate
+              certificates, creating a trust connection between a server and a client. In simpler
+              terms, it makes sure online communication is secure and trustworthy.
+            </TabPanel>
+          </TabPanels>
+        </Tabs>
+
+        <Accordion allowMultiple>
+          <Heading as="h3" size="md" marginBottom="2">
+            FAQ
+          </Heading>
+          <FaqAccordion title="How long is an SSL Certificate valid for?">
+            <Text>
+              To find out more about this, it's best to get in touch with Seneca ITS as it all
+              depends on their rules and regulations. They'll be able to provide you with the
+              necessary information in this regard. You can contact them by clicking&nbsp;
+              <Link href="https://students.senecacollege.ca/spaces/190/support/wiki/view/1473/contact-its">
+                here
+              </Link>
+            </Text>
+          </FaqAccordion>
+          <FaqAccordion title="Can I use a single SSL Certificate for multiple domains?">
+            The certificate that has been issued covers all the domains you've set up under
+            dns-records.
```

Comment:
Should it be `DNS records`? Perhaps, `in DNS record table` is clearer?

Reply:
sure I'll refer to both the instruction and the table